### DISCUSSION TITLE
How to use Copilot when connected to the Internet after opening Android Studio?

### DISCUSSION THREAD

**Discussion by abhineetchandan** [community/community#36676] [Copilot Conversations]
**The problem is as follow:**
Version: 1.1.33.1980, OS: Windows 11, IDE: Android Studio Dolphin 2021.3.1 Patch 1.
1. I open my folder in Android Studio (not connected to internet). So Github Pilot is not logged in.
2. After I connect to the Internet, I login to Github Pilot. But after logging in, suggestions are still not showing when typing.

Thanks

**Comment by iamcjwilliams** [comment]
@abhineetchandan Can you perform the following (View -> Output; select GitHub Copilot from the pulldown)?

**Comment by ossanna16** [comment]
Hey there @abhineetchandan and thank you for asking a good question! I'm glad you found our community 🙂
